src/containers/home/home.component.jsx

```diff
@@ -30,23 +57,32 @@ const Home = (props) => {
     }
   }, []);
 
-  let displayPost = [];
+  useEffect(() => {
+    console.log("hitting again");
+    if (postsFromState.posts.length > 0) {
+      const displayPost = postsFromState.posts.map((post) => {
+        let hasUserVoted = false;
+        if (userVoteHistory.includes(post.id)) {
+          hasUserVoted = true;
```

Comment:
This whole thing is better written as `const hasUserVoted = userVoteHistory.includes(post.id)`

Reply:
Done, good shout, I'll remember that syntax